examples/misc/lib/effective_dart/usage_bad.dart

```diff
@@ -1,7 +1,7 @@
 // ignore_for_file: avoid_init_to_null, empty_constructor_bodies, final_not_initialized_constructor_1, prefer_is_not_empty, sort_constructors_first, type_annotate_public_apis, type_init_formals, unnecessary_brace_in_string_interps, unnecessary_getters_setters, unused_element, unused_local_variable
 import 'dart:async';
 import 'dart:io';
-import 'dart:math';
+import 'dart:math' hide Point;
```

Comment:
We don't need to hide `Point` if we avoid adding a fourth Point class in `usage_good.dart` (see that file for details).